docs/src/components/MDXContent/MDXContent.js

```diff
@@ -20,40 +20,20 @@
 
```

Comment:
Refactored `MDXContent` to only do one thing: render MDX (separation of concerns). Adding the 'edit page' link is now handled in the same place that github, npm links etc are added.